test/shopify.test.js

```diff
@@ -215,4 +215,44 @@ describe('Shopify', () => {
         .then(res => expect(res).to.deep.equal({}));
     });
   });
+
+  describe('Shopify#callLimits', () => {
```

Comment:
Sorry for being picky but this is not a method, it's a feature in the request method.
Can you please move these test in the above describe?

Thank you!